src/Confluent.Kafka/IConsumer.cs

```diff
@@ -0,0 +1,449 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Confluent.Kafka.Serialization;
+
+namespace Confluent.Kafka
+{
+    /// <summary>
+    ///     Implements a high-level Apache Kafka consumer (with 
+    ///     key and value deserialization).
+    /// </summary>
+    public interface IConsumer<TKey, TValue> : IDisposable
```

Comment:
`Consumer` implements `IDisposable` and `Consumer<TKey, TValue>` has a field `consumer`. Therefore I think `IConsumer<TKey,TValue>` should inherit from `IDisposable`. Please correct me if I`m wrong.